sign/pairing-uri.md

```diff
@@ -2,7 +2,7 @@
 
 ## Schema ([EIP-1328](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1328.md))
 
-    uri         = "wc" ":" topic [ "@" version ][ "?" parameters ]
+    uri         = "wc" ":" "sign" "-" topic [ "@" version ][ "?" parameters ]
```

Comment:
This change breaks backward compatibility in `sign`. We cannot be sure that all Dapps will use the new sdk in the same time.

We should either handle `sign` URI with prefix and without prefix on wallets side or just don't add such prefix and assume that when there is no prefix we consume `sign` sdk.

Reply:
what if this becomes uri param instead? i.e. `?sdk=sign` if there is no param we assume it is sign

Reply:
That's why I've added Backwards Compatibility section

> This change breaks backward compatibility in `sign`. We cannot be sure that all Dapps will use the new sdk in the same time.
> 
> We should either handle `sign` URI with prefix and without prefix on wallets side or just don't add such prefix and assume that when there is no prefix we consume `sign` sdk.

Reply:
Ahh sorry @Elyniss I didn't notice